lib/open_project/translations/models/i18n_provider.rb

```diff
@@ -16,87 +16,79 @@ def create_handle
+               source: path_to_translation.to_s,
+               title: title,
+               export_pattern: export_pattern)
     end
+  rescue Crowdin::API::Errors::Error => e
+    puts "Error during update of #{@project_de}: #{e.message}"
   end
 
   def add_directory_if_missing
-    unless crowdin_directory_exists?
-      @crowdin.add_directory(@crowdin_directory)
-    end
+    @crowdin.add_directory(@crowdin_directory) unless crowdin_directory_exists?
   end
 
   def crowdin_directory_exists?
-    @crowdin.project_info['files'].find {|f| f['name'] == @crowdin_directory && f['node_type'] == 'directory'}
+    @crowdin.project_info['files'].find { |f| f['name'] == @crowdin_directory && f['node_type'] == 'directory' }
   end
 
   def file_exists_in_directory?(file)
-    directory = @crowdin.project_info['files'].find {|f| f['name'] == @crowdin_directory && f['node_type'] == 'directory'}
-    directory['files'].find {|f| f['name'] == file}
+    directory = @crowdin.project_info['files'].find { |f| f['name'] == @crowdin_directory && f['node_type'] == 'directory' }
+    directory['files'].find { |f| f['name'] == file }
   end
 
   def update_file(dest:, source:, title:, export_pattern:)
-    @crowdin.update_file [{dest: dest,
-                          source: source,
-                          title: title,
-                          export_pattern: export_pattern}],
-                          type: 'yaml'
+    @crowdin.update_file [{ dest: dest,
+                            source: source,
+                            title: title,
+                            export_pattern: export_pattern }],
+                         type: 'yaml'
   end
 
   def add_file(dest:, source:, title:, export_pattern:)
-    @crowdin.add_file [{dest: dest,
-                          source: source,
-                          title: title,
-                          export_pattern: export_pattern}],
-                          type: 'yaml'
+    @crowdin.add_file [{ dest: dest,
+                         source: source,
+                         title: title,
+                         export_pattern: export_pattern }],
+                      type: 'yaml'
   end
 
   def request_build
-    # todo maybe this could run into a timeout?
-    begin
-      @crowdin.export_translations
-    rescue Crowdin::API::Errors::Error => e
-      puts "Error during update of #{@project_id}: #{e.message}"
-    end
+    # TODO: maybe this could run into a timeout?
+
+    @crowdin.export_translations
+  rescue Crowdin::API::Errors::Error => e
+    puts "Error during update of #{@project_id}: #{e.message}"
   end
 
   def download_locales(output)
-    # todo what about errors? maybe a timeout?
-    begin
-      @crowdin.download_translation 'all', output: output
-    rescue Crowdin::API::Errors::Error => e
-      puts "Error during update of #{@project_id}: #{e.message}"
-    end
+    # TODO: what about errors? maybe a timeout?
+
+    @crowdin.download_translation 'all', output: output
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.